frontend/src/hooks/analytics/use-analytics-track.ts

```diff
@@ -0,0 +1,9 @@
+import { useCallback } from "react";
+import mixpanel, { Dict } from "mixpanel-browser";
+
+export const useAnalyticsTrack = () => {
```

Comment:
`useAnalyticsTrack` we have to use in components only?

Reply:
I didn't get the question :)
We can use it everywhere where we want track something to analytics on action.